src/components/settings/Theme.vue

```diff
@@ -1,20 +1,24 @@
 <template>
   <section>
     <h3 class="text-3xl">Appearance</h3>
-    <hr class="mt-2 mb-4">
+    <hr class="mt-2 mb-4" />
     <Extendable scope="app.settings.theme">
       <div class="mb-4">
         <div>
           <label class="button button-size-medium button-color-gray">
-            <input v-model="theme" type="radio" value="dark" class="radio">
+            <input v-model="theme" type="radio" value="auto" class="radio" />
+            <span class="ml-3">Auto</span>
+          </label>
+          <label class="button button-size-medium button-color-gray">
```

Comment:
Adding the `ml-2` class will fix the spacing here.

```suggestion
          <label class="button button-size-medium button-color-gray ml-2">
```